DeepFlux for Skeletons in the Wild/README.md

```diff
@@ -0,0 +1,99 @@
+# DeepFlux for Skeletons in the Wild
+
+[The DagsHub Repo](https://dagshub.com/Bharat-mtr/DeepFlux)
+
+Computing object skeletons in natural images is challenging, owing to large variations in object appearance and scale, and the complexity of handling background clutter. Many recent methods frame object skeleton detection as a binary pixel classification problem, which is similar in spirit to learning-based edge detection, as well as to semantic segmentation methods. In the present article, we depart from this strategy by training a CNN to predict a twodimensional vector field, which maps each scene point to a candidate skeleton pixel, in the spirit of flux-based skeletonization algorithms. This “image context flux” representation has two major advantages over previous approaches. First, it explicitly encodes the relative position of skeletal pixels to semantically meaningful entities, such as the image points in their spatial context, and hence also the implied object boundaries. Second, since the skeleton detection context is a region-based vector field, it is better able to cope with object parts of large width. We evaluate the proposed method on three benchmark datasets for skeleton detection and two for symmetry detection, achieving consistently superior performance over state-of-the-art methods.
+
+The code and trained models of:
+
+DeepFlux for Skeletons in the Wild, CVPR 2019 [[Paper]](http://openaccess.thecvf.com/content_CVPR_2019/papers/Wang_DeepFlux_for_Skeletons_in_the_Wild_CVPR_2019_paper.pdf)
+
+## Prerequisite
+
+- Caffe and VGG-16 pretrained model [[VGG_ILSVRC_16_layers.caffemodel]](http://www.robots.ox.ac.uk/~vgg/software/very_deep/caffe/VGG_ILSVRC_16_layers.caffemodel)
+
+- Datasets: [[SK-LARGE]](https://dagshub.com/Bharat-mtr/DeepFlux/src/master/data/SK-LARGE), [[SYM-PASCAL]](https://dagshub.com/Bharat-mtr/DeepFlux/src/master/data/SymPASCAL-by-KZ)
```

Comment:
As far as I can see, in the SK-Large dataset, the images folder only contained a `test` folder, isn't it missing a `train` folder too?

Reply:
Also, both datasets are in double folders, which makes it more confusing to navigate (i.e. the first dataset is under `data/SK-LARGE/SK-LARGE/`.

Would it be possible to extract them so the redundancy doesn't exist?

Reply:
Thanks for your suggestion. Some how I forgot to add the train folder. I have added it in new commit. Please check it. also I have removed redundant folders as well.

Please check it.